src/subscription/confirmation-modal/ConfirmationModal.jsx

```diff
@@ -30,19 +33,33 @@ const messages = defineMessages({
 /**
  * ConfirmationModal
  */
-export const ConfirmationModal = ({ isVisible }) => {
-  const [isOpen] = useState(isVisible);
-  const { programTitle, price, currency } = useSelector(detailsSelector);
-
+export const ConfirmationModal = () => {
+  const {
+    programTitle,
+    price,
+    currency,
+    programUuid,
+  } = useSelector(detailsSelector);
   const intl = useIntl();
-  // TODO: add the redirect URL logic for `Goto Dashboard` button
+  const { confirmationStatus } = useSelector(subscriptionStatusSelector);
+  const [isOpen, setOpen] = useState(false);
```

Comment:
I didn't see a close button on the mockup, curious if a learner is supposed to be able to close the modal if they click outside of the modal?

Reply:
As discussed in today's standup we don't want to close it even clicked outside.